src/ring_theory/ideal/operations.lean

```diff
@@ -1139,6 +1139,29 @@ end
 
 section quotient_algebra
 
+/-- The `R`-algebra structure on `R/I` -/
+instance {I : ideal R} : algebra R (ideal.quotient I) := (ideal.quotient.mk I).to_algebra
+
+/-- The canonical morphism `R →ₐ[R] I.quotient`, for `I` an ideal of `R`, as morphism of
+`R`-algebras. -/
+def quotient.mk.alg (I : ideal R) : R →ₐ[R] I.quotient :=
+⟨λ a, submodule.quotient.mk a, rfl, λ _ _, rfl, rfl, λ _ _, rfl, λ _, rfl⟩
+
+lemma quotient.mk.alg_to_ring_hom (I : ideal R) :
+  (quotient.mk.alg I).to_ring_hom = ideal.quotient.mk I := rfl
+
+lemma quotient.mk.alg_eq_mk (I : ideal R) :
+  ⇑(quotient.mk.alg I) = ideal.quotient.mk I := rfl
+
+/-- The canonical morphism `R →ₐ[R] I.quotient` is surjective. -/
+lemma quotient.mk.alg_surjective (I : ideal R) : function.surjective (quotient.mk.alg I) :=
+λ y, quotient.induction_on' y (λ x, exists.intro x rfl)
```

Comment:
This is probably just `quotient.mk_surjective` If that doesn't exist it should be added :see_no_evil:

Reply:
Yes, there is `ideal.quotient.mk_surjective`, but now is about the morphism as algebras

Reply:
You should be able to write this proof as defeq to that one at least

Reply:
Thank you, I will look at it later today.

Reply:
Indeed it works :+1: